templates/CRM/common/civicrm.settings.php.template

```diff
@@ -260,7 +260,7 @@ if (!defined('CIVICRM_UF_BASEURL')) {
  * un-comment the line describing the path. Note that this will become always enabled in
  * the near future - this opt-in setting is transitional.
  */
-if (!defined('CIVICRM_SMARTY_AUTOLOAD_PATH')) {
+if (!defined('CIVICRM_SMARTY_AUTOLOAD_PATH') && getenv('CIVICRM_SMARTY_AUTOLOAD_PATH') === FALSE)) {
```

Comment:
```suggestion
if (((((!defined('CIVICRM_SMARTY_AUTOLOAD_PATH') && ((((((getenv('CIVICRM_SMARTY_AUTOLOAD_PATH') === FALSE))))))))))) {
```

Reply:
Hmm - perhaps tat was overkill.... I thought it was the fix for the test fail but rushed